sonic-net/sonic-buildimage
# [Nokia-7220 IXR] Add support for NOKIA 7220 IXR-H4

#### Why I did it
Add support for NOKIA 7220 IXR-H4

> NOKIA 7220-IXR-H4:
> Platform: x86_64-nokia_ixr7220_h4-r0
> HwSKU: Nokia-IXR7220-H4
> ASIC: Broadcom
> Port Config: 64x400G + 2x10G

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it

1. Added module under ../device/nokia directory.
2. Modified the platform/broadcom/one-image.mk and platform/broadcom/ platform-modules-nokia.mk 
3. Update PMON submodule link

#### How to verify it

1.	Make sure the sonic-buildimage is successful 
2.	Run this image on x86_64-nokia_ixr7220_h4-r0 and verify all dockers are up and test basic commands like:
                  o	show version
                  o	show platform summary
                  o	show platform syseeprom
                  o	show platform fan
                  o	show platform psustatus
                  o	show platform firmware status
                  o	show platform temperature
                  o	sudo show system-health detail
                  o	show interface status
3.	Run OC test with T1 topology and have 100% pass on platform.



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
